### PR TITLE
Adds a versionnumber to the footer-element

### PR DESCRIPTION
**Pull request by FritzHoing**
Implements the css, so that the element is shown in the application as wanted.

## Description

Adds a div within the footer, that displays the version-number and a string "version", that can be translated via the translate function.
The css let's the version-number disappear below a max-width of 1000px.  

## Related issues or pull requests

none

## Pull request type

- [ ] Bugfix
- [x] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)

## Do you introduce a breaking change?

- [ ] Yes

- [x] No

## Checklist

- [x] I have added a screenshot/screencast to illustrate the visual output of my update.
- [ ] I have added the proposed change to the `CHANGELOG.md`.
- [x] I have run the test suite successfully (run `npm test` locally).

versionnumber

### CODE REVIEW

**Comment by dnlkoch** [comment]
Is this one ready to review?

File: src/component/container/Footer/Footer.tsx [hunk 1]
```diff
@@ -227,6 +229,9 @@ export class Footer extends React.Component<FooterProps, FooterState> {
           <span>{t('MousePositionLabel')}: </span>
           <div id="mouse-position"/>
         </div>
+        <div className="version_number">
+          <span>{t('Version')}: {version}</span>
+        </div>
```
Comment (dnlkoch) [inline src/component/container/Footer/Footer.tsx:234]: This element shouldn't be rendered if not version is given.

File: src/resources/i18n/de.json [hunk 2]
```diff
@@ -167,5 +167,6 @@
   },
   "LayerTreeApplyTimeInterval": {
     "setTimeLineToLayerInterval": "Zeitraum des Layers anwenden"
-  }
+  },
+  "Version": "Versionsnummer"
```
Comment (dnlkoch) [inline src/resources/i18n/de.json:171]:
```suggestion
  "Version": "Version"
```

File: src/component/container/Footer/Footer.tsx [hunk 3]
```diff
@@ -227,6 +229,9 @@ export class Footer extends React.Component<FooterProps, FooterState> {
           <span>{t('MousePositionLabel')}: </span>
           <div id="mouse-position"/>
         </div>
+        <div className="version_number">
+         {version && <span>{t('Version')}: {version}</span>}
+        </div>
```
Comment (FritzHoing) [inline src/component/container/Footer/Footer.tsx:234]:
```suggestion
        {version && <div className="version_number">
          <span>{t('Version')}: {version}</span>
        </div>}
```

**Review by FritzHoing — COMMENTED** [review]
makes the <div></div> conditional

**Review by dnlkoch — APPROVED** [review]

**Pull request merged**